<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneProperties</class>
 <widget class="QWidget" name="SceneProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>473</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>223</width>
        <height>453</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="11" column="1">
        <widget class="QComboBox" name="music">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The music that will be played when the stage starts&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QPushButton" name="editPalette">
         <property name="text">
          <string>Edit Palette</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QPushButton" name="editChunks">
         <property name="text">
          <string>Edit Chunks</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Layer 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="activeLayer3">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1st Active Layer (Draws on Layer 5)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Foreground</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Music</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Layer 3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="title"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Layer 0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Layer Midpoint</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="activeLayer0">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1st Active Layer (Draws on Layer 1)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Foreground</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="layerMidpoint">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimum activr layer for tiles with &amp;quot;High&amp;quot; Visual Layer to be drawn on (tiles with &amp;quot;Low&amp;quot; Visual Layer wont be drawn)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>After Layer 0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>After Layer 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>After Layer 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>After Layer 3</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="activeLayer1">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1st Active Layer (Draws on Layer 2)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Foreground</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="activeLayer2">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1st Active Layer (Draws on Layer 3)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Foreground</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Player Spawn Position</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="plrY">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player spawn YPos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="minimum">
             <number>-32767</number>
            </property>
            <property name="maximum">
             <number>32767</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="plrX">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player spawn XPos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="minimum">
             <number>-32767</number>
            </property>
            <property name="maximum">
             <number>32767</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="background">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The starting background layer that will be displayed in-game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Background 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background 8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QPushButton" name="editStageconfig">
         <property name="text">
          <string>Edit Stageconfig</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Layer 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="loadGlobals">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Load Global Objects</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
